packages/devops/scripts/deployment-common/setup.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash -le
+
+# This script is used by Amazon Image Builder to pre-bake a Tupaia AMI
+# To deploy changes, upload the latest to the S3 bucket "tupaia_devops"
+
+HOME_DIR=/home/ubuntu
+TUPAIA_DIR=$HOME_DIR/tupaia
+
+cd $HOME_DIR
+
+sudo apt-get update
+
+# install nginx and add h5bp config
+if ! command -v nginx &> /dev/null
+then
+  sudo apt-get install -yqq nginx
+  git clone https://github.com/h5bp/server-configs-nginx.git
+  cd ./server-configs-nginx
+  git checkout tags/2.0.0
+  cd ..
+  sudo cp -R ./server-configs-nginx/h5bp/ /etc/nginx/
+  rm -rf server-configs-nginx
+
+  # Add the nginx user (www-data) to the ubuntu group to give it access to the tupaia code
+  sudo usermod -a -G ubuntu www-data
+else
+  echo "nginx already installed, skipping"
+fi
+
+# install psql for use when installing mv refresh in the db
+sudo apt-get install -yqq postgresql-client
+
+# Install base dependencies
```

Comment:
Why the move to just one `sudo apt-get`? Time saving?
Also note the removal of `--no-install-recommends`

Reply:
I think the initial reasoning might be lost to time. I'll wind it back tho since it doesn't seem to be needed explicitly